### DISCUSSION TITLE
Abrir um programa .w do Totvs Datasul dentro tela POUI

### DISCUSSION THREAD

**Discussion by dalpogetto** [po-ui/po-angular#1682] [Tire suas dúvidas]
Bom dia a todos,
Alguem sabe como se abre um programa .w de dentro de uma tela de POUI?
Exemplo: colocar um botao na tela em POUI para abrir o programa FT4001 .
Alguem sabe como se faz ?

**Comment by anliben** [comment]
Bom Dia @dalpogetto, tudo bem?

Não consegui identificar sua necessidade, mas tenho umas dicas:

1 - Não disponibilizamos esta informação sobre como importar uma tabela de cálculo de embarque: FT4001
2 - Não temos uma forma nativa de importação desta e outras tabelas do TOTVS Datasul
3 - Podes criar um WebService que será consumido pelo componente po-table ou po-dynamic-table do POUI

Caso tenha respondido sua dúvida nos avise, e caso tenha interesse em compartilhar mais sobre sua necessidade pode anexar a esta discussão exemplos de como fazer!

**Comment by dalpogetto** [comment]
Oi João Victor ,
Muito obrigado pelo retorno .

Na verdade eu apenas quero abrir uma tela desenvolvida em DDK de dentro da
tela do Po-Ui.

Queria apenas abrir uma tela , como fosse um run FTP/FT4001.r .


Se conseguir me ajudar eu agradeceria muito .


Tenha um ótimo dia ,

Em seg., 10 de abr. de 2023 09:46, João Victor ***@***.***>
escreveu:
